.github/workflows/old_ci.yml

```diff
@@ -55,6 +55,20 @@ jobs:
               B2_LINKFLAGS: linkflags=-fsanitize=undefined linkflags=-fno-sanitize-recover=all linkflags=-fuse-ld=gold
             os: ubuntu-20.04
             install: g++-8
+          - name: New var usage, multiple values per key
+            env:
+              B2_CI_VERSION: 1
+              B2_TOOLSET: gcc-8
+              B2_ADDRESS_MODEL: 64
+              B2_LINK: shared,static
+              B2_THREADING: threading=multi,single
+              B2_VARIANT: release
+              # Possible (ab)usage
+              B2_CXXFLAGS: define=norecover
```

Comment:
I'd consider this as unsupported. With `B2_CI_VERSION=1` you "promise" to use the new API, i.e. the abuse of B2_CXXFLAGS here isn't allowed.